### PR TITLE
V2.24.2 support #629

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [√] Code must follow existing styling conventions
- [√] Added a descriptive commit message
- [ ] Sample apps updated if needed

### Solves issue(s)
629

### CODE REVIEW

**Comment by nicwise** [comment]
I've just done a patch for this in #657 - which can be closed. Whats the ETA on getting this merged in @marinaserranomontes @jeffswartz ?

Might be "yeah, timezones, we are asleep right now", which is fair :)

**Comment by elie-chaaban** [comment]
when are we merging this? I need it asap.

does this make the min android version to 11?

**Comment by nicwise** [comment]
Nope, that the JDK version I think

**Comment by elie-chaaban** [comment]
What's the benefit of this version? On android 9, it happens most of the time on Android that the video is black

**Comment by elie-chaaban** [comment]
@jeffswartz @marinaserranomontes @enricop89 when this would be merged

File: android/build.gradle [hunk 1]
```diff
@@ -12,8 +12,8 @@ android {
     buildToolsVersion _buildToolsVersion
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```
Comment (jeffswartz) [inline android/build.gradle:16]: I am changing the Java version based on the notes in the Android SDK docs[here](https://tokbox.com/developer/sdks/android/#-creating-your-own-app-using-the-opentok-android-sdk).

Comment (elie-chaaban) [reply android/build.gradle:16]: Yes just saw it

**Comment by nicwise** [comment]
@jeffswartz thanks for doing this - very much appreciated :)

**Comment by elie-chaaban** [comment]
I have used the changes done by @jeffswartz and appied them in a batch

I can see a huge difference in the video call performance especially in Android

**Pull request merged**